raiden/raiden_event_handler.py

```diff
@@ -308,6 +308,24 @@ def handle_contract_send_channelsettle(
     channel.settle()
 
 
+def handle_contract_send_channelsettle2(
+        raiden: RaidenService,
+        channel_settle_event: ContractSendChannelSettle,
+):
+    channel = raiden.chain.payment_channel(channel_settle_event.channel_identifier)
```

Comment:
We have to add the `payment_channel` function to the `BlockchainService`.

Reply:
Agreed.